packages/11ty/_includes/components/table-of-contents/item/grid.js

```diff
@@ -48,8 +48,10 @@ module.exports = function (eleventyConfig) {
      */
     const isPage = !!layout
 
+    const divider = pageContributorDivider ? pageContributorDivider : ' — '
```

Comment:
@geealbers a default value for this could be set when `pageContributorDivider` is destructured or here with the following statement
```suggestion
    const divider = pageContributorDivider || ' — '
```